src/main/features/claim/form/models/eligibility/eligibility.ts

```diff
@@ -33,81 +34,72 @@ export class Eligibility {
   })
   governmentDepartment?: YesNoOption
 
-  @IsIn(YesNoOption.all(), { message: ValidationErrors.YES_NO_REQUIRED, groups: [ValidationGroups.HELP_WITH_FEES] })
-  helpWithFees?: YesNoOption
-
   @IsIn(YesNoOption.all(), {
     message: ValidationErrors.YES_NO_REQUIRED,
     groups: [ValidationGroups.CLAIM_IS_FOR_TENANCY_DEPOSIT]
   })
   claimIsForTenancyDeposit?: YesNoOption
 
-  constructor (claimOnBehalf?: YesNoOption,
+  constructor (claimValue?: ClaimValue,
+               helpWithFees?: YesNoOption,
                claimantAddress?: YesNoOption,
                defendantAddress?: YesNoOption,
-               claimValue?: ClaimValue,
-               singleClaimant?: YesNoOption,
-               singleDefendant?: YesNoOption,
                eighteenOrOver?: YesNoOption,
+               validDefendant?: ValidDefendant,
+               singleClaimant?: YesNoOption,
                governmentDepartment?: YesNoOption,
-               helpWithFees?: YesNoOption,
                claimIsForTenancyDeposit?: YesNoOption) {
-    this.claimOnBehalf = claimOnBehalf
+    this.claimValue = claimValue
+    this.helpWithFees = helpWithFees
     this.claimantAddress = claimantAddress
     this.defendantAddress = defendantAddress
-    this.claimValue = claimValue
-    this.singleClaimant = singleClaimant
-    this.singleDefendant = singleDefendant
     this.eighteenOrOver = eighteenOrOver
+    this.validDefendant = validDefendant
```

Comment:
I am not sold on the name, valid means a lot and can be misinterpreted as well. For me defendant living with England and Wales is valid defendant as well, why we did not call that page object a `validDefendant` at that time ;)

Reply:
I completely agree with you but at the time of development i needed a name. As the UX has merged 2 different stories into one, i cant see a clear and obvious name. Any ideas?